cmd/config_printer/config_man.go

```diff
@@ -0,0 +1,58 @@
+/***************************************************************
+ *
+ * Copyright (C) 2024, Pelican Project, Morgridge Institute for Research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ***************************************************************/
+
+package config_printer
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+
+	"github.com/pelicanplatform/pelican/docs"
+
+	"github.com/charmbracelet/glamour"
```

Comment:
Can you clean up these imports? Rule of thumb is first any standard packages, then any general github/misc imports, and finally any pelican imports. It looks like you already figured this out in all the other places I checked :)

Reply:
Addressed it in the latest series of commits.